config/locales/en/citizens.yml

```diff
@@ -112,8 +112,11 @@ en:
         add_other_outgoings: Add another type of regular payment
       index:
         add_all_outgoings: Add payments you made from %{start_at} to %{finish_at}.
-        page_heading: Select your payments
-        select: Select from your bank statement
+        page_heading: Your regular payments
+        select_rent_or_mortgage: Add rent or mortgage payments
+        select_child_care: Add childcare costs
+        select_maintenance_out: Add maintenance payments
+        select_legal_aid: Add legal aid payments
```

Comment:
Then I'd change this to:
```
select:
  rent_or_mortgage: Add rent or mortgage payments
  child_care: Add childcare costs
  maintenance_out: Add maintenance payments
  legal_aid: Add legal aid payments
```
A group of items with the same prefix (or suffix for that matter) is a bit of a red flag to me. It usually means the items need grouping.